src/spans/awsSpan.js

```diff
@@ -56,12 +56,17 @@ export const getCurrentTransactionId = () => {
   return getSpanInfo(lambdaEvent).traceId.transactionId;
 };
 
-export const getBasicSpan = () => {
+export const isSpanIsFromAnotherInvocation = span => {
+  return (
+    !span.id.includes(span.reporterAwsRequestId) && span.parentId !== span.reporterAwsRequestId
```

Comment:
About `span.parentId !== span.reporterAwsRequestId`.
What if there is `_started` in one of the ids? Maybe we should use `includes`?

Reply:
The parentId is the request-id, not the span id, so we don't need to include there